koku/api/query_filter.py

```diff
@@ -160,10 +160,19 @@ def add(self, query_filter=None, table=None, field=None,
             if qf not in self:
                 self._filters.append(qf)
 
-    def compose(self):
-        """Compose filters into a dict for submitting to Django's ORM."""
+    def compose(self, logical_operator=None):
+        """Compose filters into a dict for submitting to Django's ORM.
+
+        Args:
+            logical_operator (str): 'and' or 'or' -- how to combine the filters.
+
+        """
         composed_query = None
         compose_dict = defaultdict(list)
+        operator = 'and'
+        if logical_operator == 'or':
+            operator = 'or'
```

Comment:
I think this could be faster/simpler to do as a mapping using the `operator` stdlib.

```suggestion
import operator
operator_map = { 'and': operator.and_, 'or': operator.or_ }
composed_query = getattr(operator_map[operator])(composed_query, or_filter)
```